tools/wptrunner/wptrunner/executors/base.py

```diff
@@ -742,8 +743,11 @@ def process_action(self, url, payload):
         except self.unimplemented_exc:
             self.logger.warning("Action %s not implemented" % action)
             self._send_message(cmd_id, "complete", "error", "Action %s not implemented" % action)
+        except self.expected_exc:
+            self.logger.debug("Action %s failed with an expected exception" % action)
+            self._send_message(cmd_id, "complete", "error")
```

Comment:
Looks like we are still sending out the same message as previous when we get a WebDriverException. So is this change non-functional change besides some log will be different? Does not seem to be the case per the commit message.

Reply:
The difference is the missing `raise` in this block compared to the final `except` block

Reply:
Thanks for the explanation.